Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/WindowsMixedRealityUtilities.cs

```diff
@@ -62,18 +63,17 @@ public static SpatialCoordinateSystem SpatialCoordinateSystem
         {
             get
             {
+                if (spatialCoordinateSystem == null && WmrUtilitiesProvider != null)
+                {
 #if ENABLE_DOTNET
-                return spatialCoordinateSystem ?? (spatialCoordinateSystem = GetSpatialCoordinateSystem(WorldManager.GetNativeISpatialCoordinateSystemPtr()));
+                    spatialCoordinateSystem = GetSpatialCoordinateSystem(WmrUtilitiesProvider.ISpatialCoordinateSystemPtr);
```

Comment:
Consider turning this into a function?

Reply:
I'm going to leave this as-is, since it's only called in one place and, although a bit confusing with all the `#ifdef`s, I'm not sure moving it into a function will help overall reading of the method.